enterprise/coderd/workspaces_test.go

```diff
@@ -1587,6 +1586,55 @@ func TestWorkspaceTagsTerraform(t *testing.T) {
 	}
 }
 
+// downloadProviders is a test helper that creates a temporary file and writes a
+// terraform CLI config file with a provider_installation stanza for coder/coder
+// using dev_overrides. It also fetches the latest provider release from GitHub
+// and extracts the binary to the temporary dir. It is the responsibility of the
```

Comment:
Love the line length alignment here 😍, too bad about the third line missing one char 🤣

Reply:
I did that specifically to annoy you ;-P